src/components/App.jsx

```diff
@@ -0,0 +1,22 @@
+import React, { useState } from 'react';
+import Display from './Display.jsx';
+import './App.scss';
+import ButtonPanel from './ButtonPanel.jsx';
+import calculate from '../logic/calculate.js';
+
+function App() {
+  const [state, setState] = useState({
+    queue: [], total: '', next: '', operation: '', target: 'total',
+  });
+  const handleClick = (btnName) => {
+    setState(calculate(state, btnName));
+  };
+  return (
+    <div id="App">
+      <Display result={state.queue.join('') || '0'} />
```

Comment:
please pass in next and total to this component ![screen](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone5/app/passToatalToDisplay.png)